components/exchange/useSearchTranslation.js

```diff
@@ -14,6 +14,7 @@ const fetchTranslation = (word, from, to) => {
 
   return fetch(url, {
     method: 'post',
+    // mode: 'no-cors',
```

Comment:
I think this is a mistake

Reply:
Okay but I think that is something you did

Reply:
Yes I remember doing it. Could you remove it from the PR?